minio/helpers.py

```diff
@@ -375,6 +375,19 @@ def is_non_empty_string(input_string):
 
     return True
 
+def is_valid_bucket_policy_name(policy):
```

Comment:
Function name `is_valid_bucket_policy_name` is misleading, since we are not validating the name of the policy.
Change the function name something like `is_valid_type_bucket_policy`.